docs/lib/content/using-npm/scripts.md

```diff
@@ -259,32 +259,69 @@ executing the scripts.  So, if your package.json has this:
 then you could run `npm start` to execute the `bar` script, which is
 exported into the `node_modules/.bin` directory on `npm install`.
 
-#### package.json vars
+#### Environment variables from `package.json`
 
-The package.json fields are tacked onto the `npm_package_` prefix. So,
-for instance, if you had `{"name":"foo", "version":"1.2.5"}` in your
-package.json file, then your package scripts would have the
-`npm_package_name` environment variable set to "foo", and the
```

Comment:
These still exist and are very load bearing

```sh
$ npm run env|grep name
npm_package_name=npm
```

Reply:
isn't this what the extended example includes (L282)?

Reply:
No the name/version entries are distinct from the config.  They are powered by entries in package.json and can't be change via config.

Reply:
the example matches the code:

https://github.com/npm/run-script/blob/795e49e5571789efc4c4ebf3f5975ec9267bdc6d/lib/package-envs.js#L21-L29

Agree. We should be more explicit. How about:

- Only `name`, `version`, `engines`, `config`, and `bin` from [`package.json`](/configuring-npm/package-json) are available. Other fields are *not* exposed as environment variables (see [RFC](https://github.com/npm/rfcs/blob/latest/accepted/0021-reduce-lifecycle-script-environment.md)). If you want to know what is in `package.json`, you can read and parse the file or use `require('./package.json')`, for example.